examples/training/object_detection_3d/waymo/train_pillars.py

```diff
@@ -11,29 +11,70 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import os
 import time
 
 import tensorflow as tf
 
-from keras_cv.datasets.waymo import build_tensors_for_augmentation
-from keras_cv.datasets.waymo import load
 from keras_cv.layers import preprocessing3d
 
 TRAINING_RECORD_PATH = (
-    "./wod-records"  # "gs://waymo_open_dataset_v_1_0_0_individual_files/training"
+    "./wod_transformed"  # "gs://waymo_open_dataset_v_1_0_0_individual_files/training"
 )
 
+global_batch = 1
+
+features_dict = {
+    "point_clouds": tf.io.RaggedFeature(dtype=tf.float32),
+    "bounding_boxes": tf.io.RaggedFeature(dtype=tf.float32),
+}
+
+
+def build_tensors(x):
```

Comment:
(not for this PR, just a thought): it might be nice to eventually bundle this in with the loader?

The loader could have two public methods: one for extracting point clouds / bboxes and serializing them into the denser tfrecords that we'll use for training, and the other for loading the dense records